### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by deangalvin-cb**
### What changed? Why?
This brings the readme inline with: https://github.com/coinbase/coinbase-sdk-nodejs/pull/190

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: README.md [hunk 1]
```diff
@@ -3,12 +3,9 @@
 
 The Coinbase Go SDK enables the simple integration of crypto into your app. By calling Coinbase's Platform APIs, the SDK allows you to provision crypto wallets, send crypto into/out of those wallets, track wallet balances, and trade crypto from one asset into another.
 
-The SDK currently supports Customer-custodied Wallets on the Base Sepolia test network.
+The SDK supports various verbs on Customer-custodied Wallets across multiple networks, as documented [here](https://docs.cdp.coinbase.com/cdp-sdk/docs/networks).
```
Comment (drohit-cb) [inline README.md:6]: nit: Is Customer-custodied Wallets a widely used term in CDP ? Can we call out the fact that this currently only works on External Addresses ?

File: README.md [hunk 2]
```diff
@@ -3,12 +3,9 @@
 
 The Coinbase Go SDK enables the simple integration of crypto into your app. By calling Coinbase's Platform APIs, the SDK allows you to provision crypto wallets, send crypto into/out of those wallets, track wallet balances, and trade crypto from one asset into another.
```
Comment (drohit-cb) [inline README.md:4]: Do we wanna remove this part "send crypto into/out of those wallets, track wallet balances," ?

**Review by drohit-cb — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (deangalvin-cb) [reply README.md]: We call it: `End-User Wallets` everywhere in the docs so I update to that!

**Review by drohit-cb — APPROVED** [review]

**Pull request merged**